Meta/run.sh

```diff
@@ -47,19 +61,30 @@ $SERENITY_EXTRA_QEMU_ARGS
 
 export SDL_VIDEO_X11_DGAMOUSE=0
 
+[ ! "${SERENITY_BUILD+is_set}" ] && {
+    if [ -d "$SERENITY_ROOT/Build" ]; then
+        SERENITY_BUILD="$SERENITY_ROOT/Build"
+    elif [ -d "$SERENITY_ROOT/build" ]; then
+        SERENITY_BUILD="$SERENITY_ROOT/build"
+    fi
+}
```

Comment:
All it takes is removing this.

Reply:
ping @bugaevc

Reply:
Are you, uh, sure the `{}` doesn't make it a subshell (and so the variables set won't persist to the outer shell)?

Reply:
Yeah, yeah, I'm sure. `()` creates a subshell; `{}` only groups commands.

Reply:
Right; my bad.